ocs_samples/basic_samples/SDS/JavaScript/NodeJs/Sample.js

```diff
@@ -427,18 +437,17 @@ var app = function (request1, response)
         }
     ).catch(function (err) { logError(err); });
 
+
```

Comment:
this extra line probably isn't necessary (sorry for the nitpicking)

Reply:
lol its good